src/Parsers/LatLongParser.php

```diff
@@ -92,24 +92,4 @@ protected function getParsers() {
 		return $parsers;
 	}
 
-	/**
-	 * Convenience function for determining if something is a valid coordinate string.
-	 * Analogous to creating an instance of the parser, parsing the string and checking isValid on the result.
-	 *
-	 * @deprecated since 2.0, please instantiate and call isValid() instead
-	 *
-	 * @param string $string
-	 *
-	 * @return boolean
-	 */
-	public static function areCoordinates( $string ) {
-		static $parser = null;
-
-		if ( $parser === null ) {
-			$parser = new self();
-		}
-
-		return $parser->parse( $string )->isValid();
```

Comment:
I just realized this isValid method does not exist, and I can not even find it in the Git history.